src/compas_rhino/geometry/brep/brep.py

```diff
@@ -440,6 +466,26 @@ def from_boolean_intersection(cls, breps_a, breps_b):
         )
         return [RhinoBrep.from_native(brep) for brep in resulting_breps]
 
+    def slice(self, plane):
+        """Slice through the Brep with a plane.
+
+        Parameters
+        ----------
+        plane : :class:`~compas.geometry.Plane` or :class:`~compas.geometry.Frame`
+            The plane to slice through the brep.
+
+        Returns
+        -------
+        list(:class:`~compas.geometry.Curve`)
+            Zero or more curves which represent the intersection(s) between the brep and the plane.
+
+        """
+        if isinstance(plane, Frame):
+            plane = Plane.from_frame(plane)
```

Comment:
is there a reason why trimming converts a plane to a frame, and slicing converts a frame to a plane?

Reply:
totally random. but changed in `trimmed` as makes more sense to eventually use `Plane`..